slides/testing/parser/file-parser.ts

```diff
@@ -0,0 +1,16 @@
+import { File } from './file';
+import { IParser } from './parser.interface';
+
+export class FileParser implements IParser<Array<File>> {
```

Comment:
The idea was to keep the `IParser` generic, so that it can accept any type of content (string, file, JSON etc..) and let the class that implements Parser take care of the actual implementation to parse the data out.